examples/with-docker/.env.example

```diff
@@ -0,0 +1,6 @@
+# Tokens
+DISCORD_TOKEN=
+OWNERS=
+
+# Configuration
+DEFAULT_PREFIX=dr!
```

Comment:
Since they are empty in the default `config.json` files in other examples, maybe keep this one empty too?